NoQuD/HomogeneousSolve/NodalSolve.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python
+
+""" Solves a homogenized problem using nodal methods."""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from NoQuD.homogenize.homogenize import *
+from NoQuD.nodal.Nodal import *
+from numpy.polynomial.legendre import legval as legval
+
+
+class NodalSolve:
+
+    """ Creates homogenized data for a 1D, 2 group problem and solves using nodal quasi-diffusion methods.
+
+    The homogenized nuclear data for the assembly(s) described by the input files is calculated. From this homogenized
+    data, the linear system required for a nodal solve is generated. Variables necessary for the solve process are
+    initialized.
+
+    Args:
+        assembly_input_files (str[]):  should contain the assembly information file, followed by the assembly information
+        files for each unique assembly.
+
+    Attributes:
+        homogenized_problem (HomogenizeGlobe instance): contains homogenized nuclear data.
```

Comment:
if homogenized_problem contains the homogenized nuclear data why not call it "homogenized_data" or something like that?